docs/documentation/siddhi-4.0.md

```diff
@@ -2199,9 +2199,10 @@ and granularity.
 
 ```sql
 from <aggregation>
+<select condition>?
```

Comment:
You have 2 selects

Reply:
This is to represent on condition(which is optional)
from TradeAggregation  * on symbol == "FB" *
We have referred to them as join condition in other places, IMO join condition does not make sense here.

Reply:
There should not be <select part